packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/Detail/AdvancedTxDetails/IODetails/TransactionDetailsModal.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import styled, { useTheme } from 'styled-components';
+
+import { getDisplaySymbol } from '@suite-common/wallet-config';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { formatAmount, formatNetworkAmount } from '@suite-common/wallet-utils';
+import { Column, NewModal, Paragraph } from '@trezor/components';
+import { borders, spacings, spacingsPx } from '@trezor/theme';
+
+import { Translation } from 'src/components/suite';
+import { useSelector } from 'src/hooks/suite/useSelector';
+
+const CodeBlock = styled.code<{ $isDarkTheme: boolean }>`
```

Comment:
We might do it with `Text` component + add new `textProps`. WDYT @seibei-iguchi ?